client.go

```diff
@@ -180,31 +183,50 @@ func runStreamedCommand(img, cmd string, getError bool) error {
 // It returns the ID of the container spawned from the image.
 // Note well: the container is NOT deleted when the given command terminates.
 func runCommandInContainer(img string, cmd []string, dst io.Writer) (string, error) {
-	return startContainer(img, cmd, true, dst)
+	id, err := createContainer(img, cmd)
+	if err != nil {
+		log.Println(err)
+		return "", err
+	}
+
+	error103 := false
+	for error103 == false {
```

Comment:
`for !error103 {`

Reply:
I prefer a more intuitive identifier, for instance `runCommand` or `execute` or something that indicates the semantics behind.

Reply:
@mssola let me explain my thoughts on this:

error103 is false, so 'startContainer' is executed the first time.
Now we assume, a 103 exitcode was returned.
So, he leaves error103 on false and startCommand will be executed again.
Now we assume, no exitcode 103 was returned.
Therefore we go in fo the default case, where error103 is set to true and startCommand won't be executed again.

If there are more zypper updates which return the exitcode 103, the for-loop runs until there is no exitcode 103 anymore.

Reply:
BTW, i changed error103 to 'execute' as @vrothberg mentioned before.
Therefore, startCommand will be executed, as long as 'execute' is 'true'.